cmd/multiFaToVcf/multiFaToVcf.go

```diff
@@ -10,14 +10,13 @@ import (
 	"log"
 )
 
-func multiFaToVcf(inFile string, chr string, outFile string) {
+func multiFaToVcf(inFile string, chr string, outFile string, substitutionsOnly bool, retainN bool) {
 	f := fasta.Read(inFile)
-	v := convert.PairwiseFaToVcf(f, chr)
-	vcfWriter := fileio.EasyCreate(outFile)
-	header := vcf.NewHeader("nameFastaSample")
-	vcf.NewWriteHeader(vcfWriter, header)
-	vcf.WriteVcfToFileHandle(vcfWriter, v)
-	vcfWriter.Close()
+	out := fileio.EasyCreate(outFile)
+	header := vcf.NewHeader("")
+	vcf.NewWriteHeader(out, header)
+	convert.PairwiseFaToVcf(f, chr, *out, substitutionsOnly, retainN)
```

Comment:
I think you will want out instead of *out here because I believe *fileio.EasyWriter implements the io.Writer interface, but fileio.EasyWriter does not (I am not sure about this and did not test it).  This also mirrors os.Create in the standard lib that returns a *File, which implements io.Writer.